BetaCameras/RealSense2/RealSense2.cs

```diff
@@ -31,6 +31,8 @@ public class RealSense2 : Camera, IDisposable
         private bool _pipelineRunning = false;
         private float _depthScale = 1.0f;
         private HashSet<string> _activeChannels = new HashSet<string>();
+        private Dictionary<string, RigidBodyTransformation> extrinsicsCache = new Dictionary<string, RigidBodyTransformation>();
```

Comment:
Why not make `Camera`'s caches protected instead of having more objects in memory?
Also, would make transition to #134 easier I think.